client/src/pages/mocks/EditMock.tsx

```diff
@@ -122,6 +122,27 @@ export const EditMock: React.FC<EditMockProps> = ({ mock, setMock }) => {
     });
   };
 
+  const onDuplicateMockResponse = (mockToDuplicate: MockResponse) => {
+    const index = mock.mockResponses.length;
+
+    return postMockResponse(snifferId as string, mockId as string, {
+      name: `${mockToDuplicate.name} copy ${index}`,
```

Comment:
I would remove the `${index}`.
The addition of copy is a standard by itself.

Reply:
ok